TicketBooking/Flight/FlightModel/FlightModel.cs

```diff
@@ -0,0 +1,48 @@
+﻿using TicketBooking.Airport;
+using TicketBooking.Class;
+using TicketBooking.Country;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicketBooking.Flight.FlightModel
+{
+    public class Flight
+    {
+        public int Id { get; set; }
+        public CountryEnum DepartureCountry { get; set; }
+        public CountryEnum DestinationCountry { get; set; }
+        public Dictionary<ClassEnum, double> Class { get; set; }
+        public string FlightNo { get; set; }
+        public DateTime DepartureDate { get; set; }
+        public AirportEnum DepartureAirport { get; set; }
+        public AirportEnum ArrivalAirport { get; set; }
+
+        public override string ToString()
+        {
+            return $"Flight ID: {Id}: Flight No {FlightNo}, departs from {DepartureCountry}, {DepartureAirport}, to {DestinationCountry}, {ArrivalAirport} at {DepartureDate.ToString()}";
+        }
+
+        public string GetFlightDetails()
```

Comment:
We need a class called `FlightPrinter` and it takes a flight and execute the functions.

Please apply to other classes.